echo-web/config/echo.yml

```diff
@@ -25,6 +33,20 @@ webhooks:
     enabled: false
     sources: []
 
+scheduler:
+  enabled: ${services.echo.cron.enabled:true}
+  threadPoolSize: 20
+  triggeringEnabled: true
+  pipelineConfigsPoller:
```

Comment:
This block is entirely already defaulted to these values in code; let's just let the code do the defaulting.